cumulus_etl/upload_notes/cli.py

```diff
@@ -208,8 +221,16 @@ def define_upload_notes_parser(parser: argparse.ArgumentParser) -> None:
     parser.add_argument(
         "--export-to", metavar="PATH", help="Where to put exported documents (default is to delete after use)"
     )
+
+    parser.add_argument(
+        "--philter",
+        choices=[PHILTER_DISABLE, PHILTER_REDACT, PHILTER_LABEL],
+        default=PHILTER_REDACT,
+        help="Whether to use philter to redact/tag PHI",
+    )
+    # Old, simpler version of the above
     parser.add_argument(
-        "--no-philter", action="store_false", dest="philter", default=True, help="Don’t run philter on notes"
+        "--no-philter", action="store_const", const=PHILTER_DISABLE, dest="philter", help=argparse.SUPPRESS
```

Comment:
very kind of you, but i think you could make a breaking change now.

Reply:
Eh, I like to give some amount of gracetime, even to ourselves and our own muscle memory. But I've added a note that the arg can be removed after May 2024, to help avoid it staying around forever.